libs/product/src/drivers/magento/queries/fragments/bundled-product.ts

```diff
@@ -1,11 +1,7 @@
 import gql from 'graphql-tag';
 
-export const bundledProductFragment = gql`
-  fragment BundledProduct on BundleProduct {
-		dynamic_price
-		dynamic_sku
-		dynamic_weight
-		ship_bundle_items
+export const magentoBundledProductFragment = gql`
+  fragment magentoBundledProduct on BundleProduct {
```

Comment:
Let's remove this from this PR, its outside scope.

Reply:
removed